mava/configs/network/mlp.yaml

```
@@ -7,7 +7,7 @@ actor_network:
     activation: relu
 
 action_head:
-  _target_: mava.networks.DiscreteActionHead # [DiscreteActionHead, ContinuousActionHead]
+  _target_: mava.networks.ContinuousActionHead # [DiscreteActionHead, ContinuousActionHead]
```

Just resetting to the default. 
```suggestion
  _target_: mava.networks.DiscreteActionHead # [DiscreteActionHead, ContinuousActionHead]
```